contracts/base/ModuleManager.sol

```diff
@@ -15,9 +55,12 @@ import {IModuleManager} from "../interfaces/IModuleManager.sol";
  * @author Stefan George - @Georgi87
  * @author Richard Meissner - @rmeissner
  */
-abstract contract ModuleManager is SelfAuthorized, Executor, GuardManager, IModuleManager {
+abstract contract ModuleManager is SelfAuthorized, Executor, IModuleManager {
     address internal constant SENTINEL_MODULES = address(0x1);
 
+    // keccak256("module_manager.module_guard.address")
```

Comment:
Should this be `// keccak256("module_manager.module_guard.address") - 1` so the pre-image is unknown and the likelihood of collision becomes super small? 

cc @nlordell that showed the trick

Reply:
We don't follow this convention for existing fallback and guard storage slots. So, did the same for Module Guard.
Do we want to change this with Module Guard?

Reply:
I have no strong opinion here; the security aspect shouldn't matter in our case since we allow delegate calls anyway. I would be curious to hear Nick's opinion.

Reply:
I wouldn't do it here, just because we don't have that convention in other storage slots that we use in the `Safe` contracts.